### ISSUE TITLE
If you don't append the input element on body on iOS it doesn't work

### ISSUE THREAD

**Issue by alessandro-bottamedi** [Jaaneek/useFilePicker#34]
`change` and `blur` event will not fire on iOS without append input element to the body.

I suggest these changes to the openFileDialog file:

**/helpers/openFileDialog.ts:**
```javascript
export function openFileDialog(accept: string, multiple: boolean, callback: (arg: Event) => void) {
  // this function must be called from a user
  // activation event (ie an onclick event)

  // Create an input element
  var inputElement = document.createElement('input');
  // Hide element and append to body (required to run on iOS safari)
  inputElement.style.display = 'none';
  document.body.appendChild(inputElement);
  // Set its type to file
  inputElement.type = 'file';
  // Set accept to the file types you want the user to select.
  // Include both the file extension and the mime type
  inputElement.accept = accept;
  // Accept multiple files
  inputElement.multiple = multiple;
  // set onchange event to call callback when user has selected file
  //inputElement.addEventListener('change', callback);
  inputElement.addEventListener('change', arg => {
    callback(arg);
    // remove element
    document.body.removeChild(inputElement);
  });
  // set onblur event to call callback when user has selected file on Safari
  inputElement.addEventListener('blur', arg => {
    callback(arg);
    // remove element
    document.body.removeChild(inputElement);
  });
  // dispatch a click event to open the file dialog
  inputElement.dispatchEvent(new MouseEvent('click'));
}
```

With this implementation everything works fine.

**Comment by Jaaneek** [comment]
Looks good
What do you think @MrKampla @GitKokoszka

**Comment by alessandro-bottamedi** [comment]
Here the PR: #35

**Comment by MrKampla** [comment]
I'm closing the issue as Your changes in the PR #35 were merged and released with version 1.4.2. Thank You for Your input.
